luma/core/framebuffer.py

```diff
@@ -89,33 +105,15 @@ class full_frame(object):
     :param device: The target device, used to determine the bounding box.
     :type device: luma.core.device.device
     """
-    def __init__(self, device):
-        self.bounding_box = (0, 0, device.width, device.height)
 
-    def redraw_required(self, image):
+    def redraw(self, image):
         """
         Caches the image ready for getting the sequence of pixel data with
```

Comment:
this docstring mentions it uses `getdata` but does it still do that?

Reply:
Reworded docs